Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,22 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json? && !args.quiet? && $stderr.tty?
+        total_formulae = if formulae_to_check == Formula
+          formulae_to_check.count
+        else
+          formulae_to_check.length
+        end
```

Comment:
```suggestion
        end

```

A little whitespace for style purposes (to make this more readable).